sale_packaging_price/views/product_view.xml

```diff
@@ -55,10 +59,12 @@
         <field name="model">product.packaging</field>
         <field name="arch" type="xml">
             <search string="Search Product Packaging">
-               <field name="product_tmpl_id"/>
-               <group expand="0" string="Group By...">
+                <field name="product_tmpl_id"/>
+                <field name="package_material"/>
+                <group expand="0" string="Group By...">
                    <filter string="Product" context="{'group_by': 'product_tmpl_id'}"/>
                    <filter string="Logistic Unit" context="{'group_by': 'ul'}"/>
+                   <filter string="Pack Type" context="{'group_by': 'package_material'}"/>
                </group>
```

Comment:
Please could you try to align all lines at the same indentation stops? Sometimes you use 3 spaces and sometimes 4, and it's a bit annoying and confusing.